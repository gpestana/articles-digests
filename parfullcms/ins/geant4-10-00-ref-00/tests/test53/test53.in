# $Id: test53.in 66892 2013-01-17 10:57:59Z gunter $
#
# Macro file for "test53.cc"
# (can be run in batch, without graphic)
#
# Lead-liquidArgon 50 layers; electron 1 GeV
#
/control/verbose 2
/run/verbose 2
#
#/testem/phys/addPhysics  empenelope
/testem/phys/addPhysics  emlivermore_pol
#/testem/phys/addPhysics  emstandard
#/testem/phys/addPhysics  emstandard_opt3
#
/run/initialize
/process/eLoss/verbose 1
#
/gun/particle gamma
/gun/energy 10 MeV
#
/testem/histo/setFileName testem3
/testem/histo/setFileType root
/testem/histo/setHisto 1 100 7. 10. MeV	#edep in absor1
/testem/histo/setHisto 2 100   0.  3. MeV	#edep in absor2
/testem/histo/setHisto 11 52   0.   52.  none	#long. profile in absor1
/testem/histo/setHisto 12 52   0.   52.  none	#long. profile in absor2
/testem/histo/setHisto 21 102  0.   102. none	#energy flow
/testem/histo/setHisto 22 102  0.   102. none	#lateral energy leakage
#
/tracking/verbose 0
/testem/event/printModulo 50
#
/run/beamOn 100
#
#/run/particle/storePhysicsTable data_stand
#/run/particle/storePhysicsTable data_pen
#/run/particle/storePhysicsTable data_liv
#


