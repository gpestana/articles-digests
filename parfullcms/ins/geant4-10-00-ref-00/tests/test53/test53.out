
        ############################################
        !!! WARNING - FPE detection is activated !!!
        ############################################

*************************************************************
 Geant4 version Name: geant4-10-00-ref-00    (6-December-2013)
                      Copyright : Geant4 Collaboration
                      Reference : NIM A 506 (2003), 250-303
                            WWW : http://cern.ch/geant4
*************************************************************

/run/verbose 2
#
#/testem/phys/addPhysics  empenelope
/testem/phys/addPhysics emlivermore_pol
#/testem/phys/addPhysics  emstandard
#/testem/phys/addPhysics  emstandard_opt3
#
/run/initialize
userDetector->Construct() start.

-------------------------------------------------------------
 ---> The calorimeter is 50 layers of:
 	        Lead:    2.3 mm 
 	 liquidArgon:    5.7 mm 
-------------------------------------------------------------

 Material: Galactic    density:  0.000 kg/m3   RadL: 204322111.300 pc   Nucl.Int.Length: 113728944.973 pc 
                       Imean:  19.200 eV   temperature:   2.73 K  pressure:   0.00 atm

   --->  Element: Galactic ( )   Z =  1.0   N =   1.0   A =   1.01 g/mole
         --->  Isotope:     1   Z =  1   N =   1   A =   1.01 g/mole   abundance:  99.99 %
         --->  Isotope:     2   Z =  1   N =   2   A =   2.01 g/mole   abundance:   0.01 %
          ElmMassFraction: 100.00 %  ElmAbundance 100.00 % 


 Material:     Lead    density: 11.350 g/cm3   RadL:   5.613 mm   Nucl.Int.Length:  18.247 cm 
                       Imean: 823.000 eV 

   --->  Element: Pb (Pb)   Z = 82.0   N = 207.2   A = 207.22 g/mole
         --->  Isotope: Pb204   Z = 82   N = 204   A = 203.97 g/mole   abundance:   1.40 %
         --->  Isotope: Pb206   Z = 82   N = 206   A = 205.97 g/mole   abundance:  24.10 %
         --->  Isotope: Pb207   Z = 82   N = 207   A = 206.98 g/mole   abundance:  22.10 %
         --->  Isotope: Pb208   Z = 82   N = 208   A = 207.98 g/mole   abundance:  52.40 %
          ElmMassFraction: 100.00 %  ElmAbundance 100.00 % 


 Material: liquidArgon    density:  1.390 g/cm3   RadL:  14.064 cm   Nucl.Int.Length:  86.022 cm 
                       Imean: 188.000 eV 

   --->  Element: Ar (Ar)   Z = 18.0   N =  40.0   A =  39.95 g/mole
         --->  Isotope:  Ar36   Z = 18   N =  36   A =  35.97 g/mole   abundance:   0.34 %
         --->  Isotope:  Ar38   Z = 18   N =  38   A =  37.96 g/mole   abundance:   0.06 %
         --->  Isotope:  Ar40   Z = 18   N =  40   A =  39.96 g/mole   abundance:  99.60 %
          ElmMassFraction: 100.00 %  ElmAbundance 100.00 % 


-------------------------------------------------------------
World is registered to the default region.
physicsList->Construct() start.
physicsList->CheckParticleList() start.
physicsList->setCut() start.
PhysicsList::SetCuts:CutLength : 1 mm 
/process/eLoss/verbose 1
#
/gun/particle gamma
/gun/energy 10 MeV
#
/testem/histo/setFileName testem3
/testem/histo/setFileType root
/testem/histo/setHisto 1 100 7. 10. MeV
----> SetHisto 1: Edep in absorber 1 (MeV)  100 bins from 7 MeV to 10 MeV
/testem/histo/setHisto 2 100 0. 3. MeV
----> SetHisto 2: Edep in absorber 2 (MeV)  100 bins from 0 MeV to 3 MeV
/testem/histo/setHisto 11 52 0. 52. none
----> SetHisto 11: longit. profile of Edep (MeV/event) in absorber 1  52 bins from 0 none to 52 none
/testem/histo/setHisto 12 52 0. 52. none
----> SetHisto 12: longit. profile of Edep (MeV/event) in absorber 2  52 bins from 0 none to 52 none
/testem/histo/setHisto 21 102 0. 102. none
----> SetHisto 21: energy flow (MeV/event)  102 bins from 0 none to 102 none
/testem/histo/setHisto 22 102 0. 102. none
----> SetHisto 22: lateral energy leak (MeV/event)  102 bins from 0 none to 102 none
#
/tracking/verbose 0
/testem/event/printModulo 50
#
/run/beamOn 100

phot:   for  gamma    SubType= 12
      LambdaPrime table from 200 keV to 10 TeV in 154 bins 
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
LivermorePolarizedPhotoElectric :  Emin=        0 eV    Emax=        1 GeV   AngularGenSauterGavrila  FluoActive
       PhotoElectric :  Emin=        1 GeV   Emax=       10 TeV   AngularGenSauterGavrila  FluoActive

compt:   for  gamma    SubType= 13
      Lambda table from 100 eV  to 1 MeV in 80 bins, spline: 1
      LambdaPrime table from 1 MeV to 10 TeV in 140 bins 
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
LivermorePolarizedCompton :  Emin=        0 eV    Emax=        1 GeV
       Klein-Nishina :  Emin=        1 GeV   Emax=       10 TeV

conv:   for  gamma    SubType= 14
      Lambda table from 1.022 MeV to 10 TeV in 140 bins, spline: 1
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
LivermorePolarizedGammaConversion :  Emin=        0 eV    Emax=        1 GeV
        BetheHeitler :  Emin=        1 GeV   Emax=       80 GeV
     BetheHeitlerLPM :  Emin=       80 GeV   Emax=       10 TeV

Rayl:   for  gamma    SubType= 11
      Lambda table from 100 eV  to 100 keV in 60 bins, spline: 0
      LambdaPrime table from 100 keV to 10 TeV in 160 bins 
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
LivermorePolarizedRayleigh :  Emin=        0 eV    Emax=        1 GeV
   LivermoreRayleigh :  Emin=        1 GeV   Emax=       10 TeV   CullenGenerator

msc:   for e-    SubType= 10
      RangeFactor= 0.04, stepLimitType: 2, latDisplacement: 1, skin= 1, geomFactor= 2.5
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
            UrbanMsc :  Emin=        0 eV    Emax=      100 MeV  Table with 120 bins Emin=    100 eV    Emax=    100 MeV
        WentzelVIUni :  Emin=      100 MeV   Emax=       10 TeV  Table with 100 bins Emin=    100 MeV   Emax=     10 TeV

CoulombScat:   for  e-    SubType= 1
      Lambda table from 100 eV  to 10 TeV in 100 bins, spline: 1
      180 < Theta(degree) < 180 pLimit(GeV^1)= 0.139531
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
  eCoulombScattering :  Emin=      100 MeV   Emax=       10 TeV

### ===  Deexcitation model UAtomDeexcitation is activated for 1 region:
          DefaultRegionForTheWorld

eIoni:   for  e-    SubType= 2
      dE/dx and range tables from 100 eV  to 10 TeV in 220 bins
      Lambda tables from threshold to 10 TeV in 220 bins, spline: 1
      finalRange(mm)= 0.1, dRoverRange= 0.2, integral: 1, fluct: 1, linLossLimit= 0.01
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
       LowEnergyIoni :  Emin=        0 eV    Emax=      100 keV
        MollerBhabha :  Emin=      100 keV   Emax=       10 TeV

eBrem:   for  e-    SubType= 3
      dE/dx and range tables from 100 eV  to 10 TeV in 220 bins
      Lambda tables from threshold to 10 TeV in 220 bins, spline: 1
      LPM flag: 1 for E > 1 GeV
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
             eBremSB :  Emin=        0 eV    Emax=        1 GeV   DipBustGen
            eBremLPM :  Emin=        1 GeV   Emax=       10 TeV   DipBustGen

msc:   for e+    SubType= 10
      RangeFactor= 0.04, stepLimitType: 2, latDisplacement: 1, skin= 1, geomFactor= 2.5
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
            UrbanMsc :  Emin=        0 eV    Emax=      100 MeV  Table with 120 bins Emin=    100 eV    Emax=    100 MeV
        WentzelVIUni :  Emin=      100 MeV   Emax=       10 TeV  Table with 100 bins Emin=    100 MeV   Emax=     10 TeV

eIoni:   for  e+    SubType= 2
      dE/dx and range tables from 100 eV  to 10 TeV in 220 bins
      Lambda tables from threshold to 10 TeV in 220 bins, spline: 1
      finalRange(mm)= 0.1, dRoverRange= 0.2, integral: 1, fluct: 1, linLossLimit= 0.01
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
        MollerBhabha :  Emin=        0 eV    Emax=       10 TeV

eBrem:   for  e+    SubType= 3
      dE/dx and range tables from 100 eV  to 10 TeV in 220 bins
      Lambda tables from threshold to 10 TeV in 220 bins, spline: 1
      LPM flag: 1 for E > 1 GeV
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
             eBremSB :  Emin=        0 eV    Emax=        1 GeV   DipBustGen
            eBremLPM :  Emin=        1 GeV   Emax=       10 TeV   DipBustGen

annihil:   for  e+    SubType= 5
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
            eplus2gg :  Emin=        0 eV    Emax=       10 TeV

CoulombScat:   for  e+    SubType= 1
      Lambda table from 100 eV  to 10 TeV in 100 bins, spline: 1
      180 < Theta(degree) < 180 pLimit(GeV^1)= 0.139531
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
  eCoulombScattering :  Emin=      100 MeV   Emax=       10 TeV

msc:   for proton    SubType= 10
      RangeFactor= 0.2, step limit type: 0, lateralDisplacement: 1, polarAngleLimit(deg)= 180
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
        WentzelVIUni :  Emin=        0 eV    Emax=       10 TeV  Table with 220 bins Emin=    100 eV    Emax=     10 TeV

hIoni:   for  proton    SubType= 2
      dE/dx and range tables from 100 eV  to 10 TeV in 220 bins
      Lambda tables from threshold to 10 TeV in 220 bins, spline: 1
      finalRange(mm)= 0.05, dRoverRange= 0.2, integral: 1, fluct: 1, linLossLimit= 0.01
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
               Bragg :  Emin=        0 eV    Emax=        2 MeV
          BetheBloch :  Emin=        2 MeV   Emax=       10 TeV

hBrems:   for  proton    SubType= 3
      dE/dx and range tables from 100 eV  to 10 TeV in 220 bins
      Lambda tables from threshold to 10 TeV in 220 bins, spline: 1
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
               hBrem :  Emin=        0 eV    Emax=       10 TeV
      ===== Limit on energy threshold has been applied 

hPairProd:   for  proton    SubType= 4
      dE/dx and range tables from 100 eV  to 10 TeV in 220 bins
      Lambda tables from threshold to 10 TeV in 220 bins, spline: 1
      Sampling table 13x1001 from 7.50618 GeV to 10 TeV 
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
           hPairProd :  Emin=        0 eV    Emax=       10 TeV

nuclearStopping:   for  proton    SubType= 8
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
   ICRU49NucStopping :  Emin=        0 eV    Emax=        1 MeV

msc:   for GenericIon    SubType= 10
      RangeFactor= 0.2, stepLimitType: 0, latDisplacement: 0
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
            UrbanMsc :  Emin=        0 eV    Emax=       10 TeV

ionIoni:   for  GenericIon    SubType= 2
      dE/dx and range tables from 100 eV  to 10 TeV in 220 bins
      Lambda tables from threshold to 10 TeV in 220 bins, spline: 1
      finalRange(mm)= 0.001, dRoverRange= 0.1, integral: 1, fluct: 1, linLossLimit= 0.02
      Stopping Power data for 17 ion/material pairs 
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
         ParamICRU73 :  Emin=        0 eV    Emax=       10 TeV

nuclearStopping:   for  GenericIon    SubType= 8
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
   ICRU49NucStopping :  Emin=        0 eV    Emax=       10 TeV

msc:   for alpha    SubType= 10
      RangeFactor= 0.2, stepLimitType: 0, latDisplacement: 0
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
            UrbanMsc :  Emin=        0 eV    Emax=       10 TeV  Table with 220 bins Emin=    100 eV    Emax=     10 TeV

ionIoni:   for  alpha    SubType= 2
      dE/dx and range tables from 100 eV  to 10 TeV in 220 bins
      Lambda tables from threshold to 10 TeV in 220 bins, spline: 1
      finalRange(mm)= 0.01, dRoverRange= 0.1, integral: 1, fluct: 1, linLossLimit= 0.02
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
            BraggIon :  Emin=        0 eV    Emax=   7.9452 MeV
          BetheBloch :  Emin=   7.9452 MeV   Emax=       10 TeV

nuclearStopping:   for  alpha    SubType= 8
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
   ICRU49NucStopping :  Emin=        0 eV    Emax=       10 TeV

msc:   for anti_proton    SubType= 10
      RangeFactor= 0.2, step limit type: 0, lateralDisplacement: 1, polarAngleLimit(deg)= 180
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
        WentzelVIUni :  Emin=        0 eV    Emax=       10 TeV  Table with 220 bins Emin=    100 eV    Emax=     10 TeV

hIoni:   for  anti_proton    SubType= 2
      dE/dx and range tables from 100 eV  to 10 TeV in 220 bins
      Lambda tables from threshold to 10 TeV in 220 bins, spline: 1
      finalRange(mm)= 0.05, dRoverRange= 0.2, integral: 1, fluct: 1, linLossLimit= 0.01
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
            ICRU73QO :  Emin=        0 eV    Emax=        2 MeV
          BetheBloch :  Emin=        2 MeV   Emax=       10 TeV

hBrems:   for  anti_proton    SubType= 3
      dE/dx and range tables from 100 eV  to 10 TeV in 220 bins
      Lambda tables from threshold to 10 TeV in 220 bins, spline: 1
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
               hBrem :  Emin=        0 eV    Emax=       10 TeV
      ===== Limit on energy threshold has been applied 

hPairProd:   for  anti_proton    SubType= 4
      dE/dx and range tables from 100 eV  to 10 TeV in 220 bins
      Lambda tables from threshold to 10 TeV in 220 bins, spline: 1
      Sampling table 13x1001 from 7.50618 GeV to 10 TeV 
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
           hPairProd :  Emin=        0 eV    Emax=       10 TeV

nuclearStopping:   for  anti_proton    SubType= 8
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
   ICRU49NucStopping :  Emin=        0 eV    Emax=        1 MeV

msc:   for kaon+    SubType= 10
      RangeFactor= 0.2, step limit type: 0, lateralDisplacement: 1, polarAngleLimit(deg)= 180
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
        WentzelVIUni :  Emin=        0 eV    Emax=       10 TeV  Table with 220 bins Emin=    100 eV    Emax=     10 TeV

hIoni:   for  kaon+    SubType= 2
      dE/dx and range tables from 100 eV  to 10 TeV in 220 bins
      Lambda tables from threshold to 10 TeV in 220 bins, spline: 1
      finalRange(mm)= 0.05, dRoverRange= 0.2, integral: 1, fluct: 1, linLossLimit= 0.01
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
               Bragg :  Emin=        0 eV    Emax=  1.05231 MeV
          BetheBloch :  Emin=  1.05231 MeV   Emax=       10 TeV

hBrems:   for  kaon+    SubType= 3
      dE/dx and range tables from 100 eV  to 10 TeV in 220 bins
      Lambda tables from threshold to 10 TeV in 220 bins, spline: 1
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
               hBrem :  Emin=        0 eV    Emax=       10 TeV
      ===== Limit on energy threshold has been applied 

hPairProd:   for  kaon+    SubType= 4
      dE/dx and range tables from 100 eV  to 10 TeV in 220 bins
      Lambda tables from threshold to 10 TeV in 220 bins, spline: 1
      Sampling table 14x1001 from 3.94942 GeV to 10 TeV 
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
           hPairProd :  Emin=        0 eV    Emax=       10 TeV

msc:   for kaon-    SubType= 10
      RangeFactor= 0.2, step limit type: 0, lateralDisplacement: 1, polarAngleLimit(deg)= 180
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
        WentzelVIUni :  Emin=        0 eV    Emax=       10 TeV  Table with 220 bins Emin=    100 eV    Emax=     10 TeV

hIoni:   for  kaon-    SubType= 2
      dE/dx and range tables from 100 eV  to 10 TeV in 220 bins
      Lambda tables from threshold to 10 TeV in 220 bins, spline: 1
      finalRange(mm)= 0.05, dRoverRange= 0.2, integral: 1, fluct: 1, linLossLimit= 0.01
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
            ICRU73QO :  Emin=        0 eV    Emax=  1.05231 MeV
          BetheBloch :  Emin=  1.05231 MeV   Emax=       10 TeV

hBrems:   for  kaon-    SubType= 3
      dE/dx and range tables from 100 eV  to 10 TeV in 220 bins
      Lambda tables from threshold to 10 TeV in 220 bins, spline: 1
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
               hBrem :  Emin=        0 eV    Emax=       10 TeV
      ===== Limit on energy threshold has been applied 

hPairProd:   for  kaon-    SubType= 4
      dE/dx and range tables from 100 eV  to 10 TeV in 220 bins
      Lambda tables from threshold to 10 TeV in 220 bins, spline: 1
      Sampling table 14x1001 from 3.94942 GeV to 10 TeV 
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
           hPairProd :  Emin=        0 eV    Emax=       10 TeV

msc:   for mu+    SubType= 10
      RangeFactor= 0.2, step limit type: 0, lateralDisplacement: 1, polarAngleLimit(deg)= 180
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
        WentzelVIUni :  Emin=        0 eV    Emax=       10 TeV  Table with 220 bins Emin=    100 eV    Emax=     10 TeV

muIoni:   for  mu+    SubType= 2
      dE/dx and range tables from 100 eV  to 10 TeV in 220 bins
      Lambda tables from threshold to 10 TeV in 220 bins, spline: 1
      finalRange(mm)= 0.05, dRoverRange= 0.2, integral: 1, fluct: 1, linLossLimit= 0.01
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
               Bragg :  Emin=        0 eV    Emax=      200 keV
          BetheBloch :  Emin=      200 keV   Emax=        1 GeV
        MuBetheBloch :  Emin=        1 GeV   Emax=       10 TeV

muBrems:   for  mu+    SubType= 3
      dE/dx and range tables from 100 eV  to 10 TeV in 220 bins
      Lambda tables from threshold to 10 TeV in 220 bins, spline: 1
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
              MuBrem :  Emin=        0 eV    Emax=       10 TeV
      ===== Limit on energy threshold has been applied 

muPairProd:   for  mu+    SubType= 4
      dE/dx and range tables from 100 eV  to 10 TeV in 220 bins
      Lambda tables from threshold to 10 TeV in 220 bins, spline: 1
      Sampling table 17x1001 from 1 GeV to 10 TeV 
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
          muPairProd :  Emin=        0 eV    Emax=       10 TeV

CoulombScat:   for  mu+    SubType= 1
      Lambda table from 100 eV  to 10 TeV in 108 bins, spline: 1
      180 < Theta(degree) < 180 pLimit(GeV^1)= 0.139531
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
  eCoulombScattering :  Emin=        0 eV    Emax=       10 TeV

msc:   for mu-    SubType= 10
      RangeFactor= 0.2, step limit type: 0, lateralDisplacement: 1, polarAngleLimit(deg)= 180
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
        WentzelVIUni :  Emin=        0 eV    Emax=       10 TeV  Table with 220 bins Emin=    100 eV    Emax=     10 TeV

muIoni:   for  mu-    SubType= 2
      dE/dx and range tables from 100 eV  to 10 TeV in 220 bins
      Lambda tables from threshold to 10 TeV in 220 bins, spline: 1
      finalRange(mm)= 0.05, dRoverRange= 0.2, integral: 1, fluct: 1, linLossLimit= 0.01
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
            ICRU73QO :  Emin=        0 eV    Emax=      200 keV
          BetheBloch :  Emin=      200 keV   Emax=        1 GeV
        MuBetheBloch :  Emin=        1 GeV   Emax=       10 TeV

muBrems:   for  mu-    SubType= 3
      dE/dx and range tables from 100 eV  to 10 TeV in 220 bins
      Lambda tables from threshold to 10 TeV in 220 bins, spline: 1
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
              MuBrem :  Emin=        0 eV    Emax=       10 TeV
      ===== Limit on energy threshold has been applied 

muPairProd:   for  mu-    SubType= 4
      dE/dx and range tables from 100 eV  to 10 TeV in 220 bins
      Lambda tables from threshold to 10 TeV in 220 bins, spline: 1
      Sampling table 17x1001 from 1 GeV to 10 TeV 
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
          muPairProd :  Emin=        0 eV    Emax=       10 TeV

CoulombScat:   for  mu-    SubType= 1
      Lambda table from 100 eV  to 10 TeV in 108 bins, spline: 1
      180 < Theta(degree) < 180 pLimit(GeV^1)= 0.139531
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
  eCoulombScattering :  Emin=        0 eV    Emax=       10 TeV

msc:   for pi+    SubType= 10
      RangeFactor= 0.2, step limit type: 0, lateralDisplacement: 1, polarAngleLimit(deg)= 180
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
        WentzelVIUni :  Emin=        0 eV    Emax=       10 TeV  Table with 220 bins Emin=    100 eV    Emax=     10 TeV

hIoni:   for  pi+    SubType= 2
      dE/dx and range tables from 100 eV  to 10 TeV in 220 bins
      Lambda tables from threshold to 10 TeV in 220 bins, spline: 1
      finalRange(mm)= 0.05, dRoverRange= 0.2, integral: 1, fluct: 1, linLossLimit= 0.01
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
               Bragg :  Emin=        0 eV    Emax=  297.505 keV
          BetheBloch :  Emin=  297.505 keV   Emax=       10 TeV

hBrems:   for  pi+    SubType= 3
      dE/dx and range tables from 100 eV  to 10 TeV in 220 bins
      Lambda tables from threshold to 10 TeV in 220 bins, spline: 1
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
               hBrem :  Emin=        0 eV    Emax=       10 TeV
      ===== Limit on energy threshold has been applied 

hPairProd:   for  pi+    SubType= 4
      dE/dx and range tables from 100 eV  to 10 TeV in 220 bins
      Lambda tables from threshold to 10 TeV in 220 bins, spline: 1
      Sampling table 16x1001 from 1.11656 GeV to 10 TeV 
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
           hPairProd :  Emin=        0 eV    Emax=       10 TeV

msc:   for pi-    SubType= 10
      RangeFactor= 0.2, step limit type: 0, lateralDisplacement: 1, polarAngleLimit(deg)= 180
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
        WentzelVIUni :  Emin=        0 eV    Emax=       10 TeV  Table with 220 bins Emin=    100 eV    Emax=     10 TeV

hIoni:   for  pi-    SubType= 2
      dE/dx and range tables from 100 eV  to 10 TeV in 220 bins
      Lambda tables from threshold to 10 TeV in 220 bins, spline: 1
      finalRange(mm)= 0.05, dRoverRange= 0.2, integral: 1, fluct: 1, linLossLimit= 0.01
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
            ICRU73QO :  Emin=        0 eV    Emax=  297.505 keV
          BetheBloch :  Emin=  297.505 keV   Emax=       10 TeV

hBrems:   for  pi-    SubType= 3
      dE/dx and range tables from 100 eV  to 10 TeV in 220 bins
      Lambda tables from threshold to 10 TeV in 220 bins, spline: 1
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
               hBrem :  Emin=        0 eV    Emax=       10 TeV
      ===== Limit on energy threshold has been applied 

hPairProd:   for  pi-    SubType= 4
      dE/dx and range tables from 100 eV  to 10 TeV in 220 bins
      Lambda tables from threshold to 10 TeV in 220 bins, spline: 1
      Sampling table 16x1001 from 1.11656 GeV to 10 TeV 
      ===== EM models for the G4Region  DefaultRegionForTheWorld ======
           hPairProd :  Emin=        0 eV    Emax=       10 TeV

Region <DefaultRegionForTheWorld> --  -- appears in <World> world volume
 This region is in the mass world.
 Root logical volume(s) : World 
 Pointers : G4VUserRegionInformation[0], G4UserLimits[0], G4FastSimulationManager[0], G4UserSteppingAction[0]
 Materials : Galactic Lead liquidArgon 
 Production cuts :   gamma 1 mm      e- 1 mm      e+ 1 mm  proton 1 mm 

Region <DefaultRegionForParallelWorld> --  -- is not associated to any world.
 Root logical volume(s) : 
 Pointers : G4VUserRegionInformation[0], G4UserLimits[0], G4FastSimulationManager[0], G4UserSteppingAction[0]
 Materials : 
 Production cuts :   gamma 1 mm      e- 1 mm      e+ 1 mm  proton 1 mm 

========= Table of registered couples ==============================

Index : 0     used in the geometry : Yes
 Material : Galactic
 Range cuts        :  gamma  1 mm     e-  1 mm     e+  1 mm  proton 1 mm 
 Energy thresholds :  gamma  250 eV     e-  250 eV     e+  250 eV  proton 100 keV
 Region(s) which use this couple : 
    DefaultRegionForTheWorld

Index : 1     used in the geometry : Yes
 Material : Lead
 Range cuts        :  gamma  1 mm     e-  1 mm     e+  1 mm  proton 1 mm 
 Energy thresholds :  gamma  102.04 keV    e-  1.36653 MeV    e+  1.28908 MeV proton 100 keV
 Region(s) which use this couple : 
    DefaultRegionForTheWorld

Index : 2     used in the geometry : Yes
 Material : liquidArgon
 Range cuts        :  gamma  1 mm     e-  1 mm     e+  1 mm  proton 1 mm 
 Energy thresholds :  gamma  6.21205 keV    e-  344.316 keV    e+  336.864 keV proton 100 keV
 Region(s) which use this couple : 
    DefaultRegionForTheWorld

====================================================================

Start closing geometry.
G4GeometryManager::ReportVoxelStats -- Voxel Statistics

    Total memory consumed for geometry optimisation:   3 kByte
    Total CPU time elapsed for geometry optimisation: 0 seconds

    Voxelisation: top CPU users:
    Percent   Total CPU    System CPU       Memory  Volume
    -------   ----------   ----------     --------  ----------
       0.00         0.00         0.00            3k Calorimeter
       0.00         0.00         0.00            0k Layer

    Voxelisation: top memory users:
    Percent     Memory      Heads    Nodes   Pointers    Total CPU    Volume
    -------   --------     ------   ------   --------   ----------    ----------
      92.05          2k         1       50         50         0.00    Calorimeter
       7.95          0k         1        3          4         0.00    Layer
### Run 0 starts.
### Run 0 start.

--------- Ranecu engine status ---------
 Initial seed (index) = 0
 Current couple of seeds = 9876, 54321
----------------------------------------

---> Begin Of Event: 0

---> Begin Of Event: 50
Run terminated.
Run Summary
  Number of events processed : 100
  User=0.14s Real=0.13s Sys=0s

------------------------------------------------------------
      material   Edep       RMS         sqrt(E0(GeV))*rmsE/Emean     total tracklen 
 
          Lead: 8.0989 MeV :  1.611 MeV      1.99 +- 0.199 %      5.99 mm  +-  1.3 mm 
   liquidArgon: 1.6766 MeV :  1.672 MeV     9.971 +- 0.9971 %      8.34 mm  +- 8.55 mm 

------------------------------------------------------------
 Beam particle gamma  E = 10 MeV

 Energy deposition from Energy flow balance : 
  material 	 Total Edep 
 
      Lead:	 8.09886 MeV
liquidArgon:	 1.67656 MeV

------------------------------------------------------------


--------- Ranecu engine status ---------
 Initial seed (index) = 0
 Current couple of seeds = 1550707139, 1349705094
----------------------------------------
#
#/run/particle/storePhysicsTable data_stand
#/run/particle/storePhysicsTable data_pen
#/run/particle/storePhysicsTable data_liv
#
G4 kernel has come to Quit state.
UserDetectorConstruction deleted.
UserPhysicsList deleted.
UserRunAction deleted.
UserPrimaryGenerator deleted.
RunManager is deleting RunManagerKernel.
EventManager deleted.
Units table cleared.
================== Deleting memory pools ===================
Number of memory pools allocated: 10 of which, static: 0
Dynamic pools deleted: 10 / Total memory freed: 0.29 Mb
============================================================
G4Allocator objects are deleted.
UImanager deleted.
StateManager deleted.
RunManagerKernel is deleted. Good bye :)
